src/cmd/ship.go

```diff
@@ -106,12 +108,18 @@ func getShipStepList(config shipConfig) (result steps.StepList) {
 	result.Append(&steps.MergeBranchStep{BranchName: branchToMergeInto})
 	result.Append(&steps.EnsureHasShippableChangesStep{BranchName: config.BranchToShip})
 	result.Append(&steps.CheckoutBranchStep{BranchName: branchToMergeInto})
-	result.Append(&steps.SquashMergeBranchStep{BranchName: config.BranchToShip, CommitMessage: commitMessage})
+	canUseDriver := canUseDriver(config.BranchToShip, branchToMergeInto)
```

Comment:
Thoughts on updating the method name to `getCanUseDriver`? I'd rather not mention API in the variable name and be consistent with only using driver